miso-web/src/main/webapp/scripts/list_completion.js

```diff
@@ -22,12 +22,12 @@
  */
 
 ListTarget.completion = {
```

Comment:
This should get renamed to `sequencingordercompletion` for symmetry with everything else.

Reply:
fixed